src/bindgen.rs

```diff
@@ -1,16 +1,40 @@
 use emoji;
 use error::Error;
 use progressbar::Step;
-use std::process::Command;
+use std::{fs, process::Command};
 use PBAR;
 
-pub fn cargo_install_wasm_bindgen(step: &Step) -> Result<(), Error> {
+static LOCAL_BINDGEN_PATH: &str = "bin/wasm-bindgen";
+
+pub fn wasm_bindgen_locally_installed(_path: &str) -> bool {
+    fs::metadata(LOCAL_BINDGEN_PATH).is_ok()
+}
```

Comment:
Should I create an absolute path here? Or is the relative path in `LOCAL_BINDGEN_PATH` enough? I think I saw an issue regarding what happens when `wasm-pack` is run from within the `pkg` directory, so I was thinking that using the crate path could make this a little less fragile.

If relative path is satisfactory, I can remove the parameter here, otherwise an absolute path can be formed with `format!` :)